webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -112,7 +113,7 @@ class SubscriptionsPage extends Component {
         const message = (
           <span>
             <span>
-              {__(`Task ${task.humanized.action} completed with a result of ${task.result}.`) + ' '}
+              {`${__(`Task ${task.humanized.action} completed with a result of ${task.result}.`)} `}
```

Comment:
I think that the translation wouldn't work as intended 
it would be better use `Jed.sprintf`

Reply:
Why not? What do I miss here?
I actually didn't change nothing here, I kept the same code and prettier did the change.